tasks/Resources/Provisioning/Hive/disconnect.robot

```diff
@@ -0,0 +1,15 @@
+*** Keywords ***
+ Disconnect Cluster
+    Oc Create    kind=ConfigMap    namespace=openshift-config
+    ...    src=tasks/Resources/Provisioning/Disconnect/disconnect.yaml
+    ...    template_data=${infrastructure_configurations}
+    Oc Patch     kind=Image    name=cluster    api_version=config.openshift.io/v1 
+    ...    src={"spec":{"additionalTrustedCA":{"name":"registry-config"}}}
+    Oc Delete   kind=Secret   name=pull-secret    namespace=openshift-config
+    Oc Create    kind=Secret    namespace=openshift-config
+    ...    src=tasks/Resources/Provisioning/Disconnect/registry_pull_secret.yaml
+    ...    template_data=${infrastructure_configurations}
```

Comment:
same template for cluster creation and registry secret creation?

Reply:
Different variables but all included in the same yaml file devoted to infrastructure variables. To use this yaml file was decided by the team some weeks ago